src/itemProcessingUtils.js

```diff
@@ -0,0 +1,69 @@
+function removeInvalidOptions(option) {
+  return Object.keys(option).includes('value', 'contemnt') &&
+    option.content != null;
+}
+
+function getOptionsFromDataSource(inputOptions, data) {
+  const {jsonData, key, value, dataName} = inputOptions;
+
+  const convertToSelectOptions = option => ({
+    value: option[key || 'value'],
+    content: option[value || 'content'],
+  });
+
+  try {
+    if (jsonData) {
+      return JSON.parse(jsonData)
+        .map(convertToSelectOptions)
+        .filter(removeInvalidOptions);
+    }
+
+    return data[dataName]
+      .map(convertToSelectOptions)
+      .filter(removeInvalidOptions);
+  } catch (error) {
+    return [];
+  }
+}
+
+function processFormItem(item) {
+  if (item.component !== 'FormMultiColumn') {
+    return item;
+  }
+
+  return item.items.flatMap(processFormItem);
+}
+
+export function getItemsFromConfig(config) {
+  return config
+    .flatMap(page => page.items)
+    .flatMap(processFormItem);
+}
+
+export function getDefaultValueForItem(item) {
+  let defaultValue = null;
+
+  if (['FormInput', 'FormTextArea', 'FormText'].includes(item.component)) {
+    defaultValue = '';
+  }
+
+  if (['FormSelect', 'FormRadioButtonGroup'].includes(item.component) && item.config.options) {
+    const options = getOptionsFromDataSource(item.config.options, this.transientData);
```

Comment:
Think I had an issue with `this.transientData`

Reply:
Looks like it relates to the radio button group

Reply:
@lisa-fehr in what way?

Reply:
@lisa-fehr can you post complete reproduction steps and I'll take a look into it?

Reply:
@JCook21 in that this.transientData is from the old file.

It says
```
TypeError: Cannot read property 'transientData' of undefined
    at getDefaultValueForItem (itemProcessingUtils.js?504e:51)
```

When I add a select or radio group to screen builder after giving them variable names and clicking preview, I get that error above.

Reply:
Ahh, good catch. Fixed. :)

Reply:
For radio group, you must select the Data Type to see the error.